src/WebJobs.Script.WebHost/Security/Key.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public class Key : IEquatable<Key>
```

Comment:
Might consider a [DebuggerDisplay] annotation to make these things easy to see in the debugger. E.g., like I've done for FunctionDescriptor.